gaia-header.js

```diff
@@ -829,6 +840,26 @@ module.exports = component.register('gaia-header', {
       inherit))));
   }
 
+  /**
+   * [ignoreDir]
+   *
+   * When the <gaia-header> component has an [ignoreDir] attribute, header
+   * direction is forced to LTR but we still want the <h1> text to be reversed
+   * to that strings like '1 selected' become 'selected 1'.
+   *
+   * When we're happy for <gaia-header> to be fully RTL responsive we won't need
+   * these rules anymore, but this depends on all Gaia apps being ready.
+   */
```

Comment:
I *guess* [bug 1179459](https://bugzilla.mozilla.org/show_bug.cgi?id=1179459) could be referenced here.